chart/kubeapps/values.yaml

```diff
@@ -1417,20 +1417,24 @@ clusters:
   - name: default
     domain: cluster.local
 
-## @skip featureFlags Feature flags (used to switch on development features)
-##
-featureFlags:
-  ## @param featureFlags.operators Enable ingress record generation for Kubeapps
-  ##
-  operators: false
-
 ## RBAC configuration
 ##
 rbac:
   ## @param rbac.create Specifies whether RBAC resources should be created
   ##
   create: true
 
+## @section Feature flags
+
+## Used enable some opt-in development features
+##
+featureFlags:
+  ## @param featureFlags.operators Enable support for Operators in Kubeapps
+  operators: false
+  ## @param featureFlags.schemaEditor Enable a visual editor for customizing the package schemas
+  ##
+  schemaEditor: false
```

Comment:
Mmm, good point. I don't know, perhaps yes, there might be more options, but this would mean this feature is no longer "in development", but is something more "stable".
I'd rather keep it under a feature flag for now, so that we can perform backward incompatible changes if needed in the future.

Reply:
Oh sorry - I wasn't proposing moving it from `featureFlags`, but just giving it structure rather than a bool (ie. the full path would be `featureFlags.schemaEditor.enabled = true`).

But fine either way, it's in development anyway :)

Reply:
Ahhh, got it, got it. I didn't get your point :P Then, yes, it could be a good idea; I can perform the changes in the main https://github.com/vmware-tanzu/kubeapps/pull/5530 PR